src/client/components/robot/model.ts

```diff
@@ -1,4 +1,15 @@
+import throttle = require('lodash.throttle')
```

Comment:
Lodash exports native ES modules. Using the default import style (`import * as throttle from 'lodash.throttle'`), TypeScript complains:

![image](https://user-images.githubusercontent.com/5924865/59913722-08613080-945c-11e9-948d-c00db30372fc.png)


Switching to `import throttle = require('lodash.throttle')` compiles, but `tslint` complains:

```
ERROR: src/client/components/robot/model.ts:1:19 - require() style import is forbidden
```

We could enable `esModuleInterop` in `tsconfig.json` or make `tslint` ignore this line.

Reply:
Fixed by switching to another package: `throttle-debounce`.